src/scripts/itemDiscuss/dimItemDiscussService.factory.js

```diff
@@ -0,0 +1,18 @@
+import angular from 'angular';
+
+angular.module('dimApp')
+  .factory('dimItemDiscussService', ItemDiscussService);
+
+function ItemDiscussService($rootScope) {
+  return {
+    dialogOpen: false,
+    addItemToDiscuss: addItemToDiscuss
+  };
```

Comment:
It seems like we either need a shared service object for this state, *or* we can use broadcast events. One or the other.